jelares/portfolio/src/main/webapp/comments/commentFetcher.js

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Fetches comments and adds them to the DOM. */
+function getComments() {
+  const numCommentsElement = document.getElementById('num-comments');
+  const numComments = numCommentsElement.value;
+
+  /* This is the part about the query string which I mentioned in the sync */
```

Comment:
@ykjchen This was what I was mentioning when I said I wasn't sure if I was implementing it correcty

Reply:
There were no alternatives I had in mind in particular, if it looks good to you it works for me too